app/packages/assistant_api/doc/PublishCatalogItemRequest.md

```
@@ -10,7 +10,7 @@ Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
 **description** | **String** |  | [optional] 
 **name** | **String** |  | 
-**resourceType** | **String** | `\"skill\"`, `\"template\"`, or `\"interface\"`. | 
+**resourceType** | **String** | Resource type: `\"skill\"`, `\"template\"`, or `\"interface\"`. | 
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)